src/components/Categories.vue

```diff
@@ -0,0 +1,128 @@
+<script setup></script>
+
+<template>
+  <div class="sidebar_infoblock">
+    <h3 class="sidebar_infoblock_name">POPULAR TAGS</h3>
+    <ul class="sidebar_infoblock_list">
+      <li class="sidebar_infoblock_list_option">
+        <a class="sidebar_infoblock_list_option_link" href="#"
+          ><div class="sidebar_infoblock_list_option_link_div">
+            PHOTOGRAPHY
+            <span class="sidebar_infoblock_list_option_link_span">26</span>
+          </div></a
+        >
+      </li>
+      <div class="sidebar_infoblock_list_line"></div>
+      <li class="sidebar_infoblock_list_option">
+        <a class="sidebar_infoblock_list_option_link" href="#"
+          ><div class="sidebar_infoblock_list_option_link_div">
+            EQUIPMENT
+            <span class="sidebar_infoblock_list_option_link_span">54</span>
+          </div></a
+        >
+      </li>
+      <div class="sidebar_infoblock_list_line"></div>
+      <li class="sidebar_infoblock_list_option">
+        <a class="sidebar_infoblock_list_option_link" href="#"
+          ><div class="sidebar_infoblock_list_option_link_div">
+            TRAVEL
+            <span class="sidebar_infoblock_list_option_link_span">17</span>
+          </div></a
+        >
+      </li>
+      <div class="sidebar_infoblock_list_line"></div>
+      <li class="sidebar_infoblock_list_option">
+        <a class="sidebar_infoblock_list_option_link" href="#"
+          ><div class="sidebar_infoblock_list_option_link_div">
+            FASHION
+            <span class="sidebar_infoblock_list_option_link_span">19</span>
+          </div></a
+        >
+      </li>
+      <div class="sidebar_infoblock_list_line"></div>
+      <li class="sidebar_infoblock_list_option">
+        <a class="sidebar_infoblock_list_option_link" href="#"
+          ><div class="sidebar_infoblock_list_option_link_div">
+            FOOD
+            <span class="sidebar_infoblock_list_option_link_span">67</span>
+          </div></a
+        >
+      </li>
+      <div class="sidebar_infoblock_list_line"></div>
+      <li class="sidebar_infoblock_list_option">
+        <a class="sidebar_infoblock_list_option_link" href="#"
+          ><div class="sidebar_infoblock_list_option_link_div">
+            MACRO
+            <span class="sidebar_infoblock_list_option_link_span">67</span>
+          </div></a
+        >
+      </li>
+    </ul>
+  </div>
+</template>
+
+<style scoped>
+* {
```

Comment:
I thnk that should be moved to App.vue and used there as there are global styles which are not scoped